### ISSUE TITLE
Code in exercises is "very low contrast" error in accessibility checker

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/workbench#76]
this might not be the right place, but while doing [accessibility checks on an incubator lesson](https://wave.webaim.org/report#/https://carpentries-incubator.github.io/deep-learning-intro/3-monitor-the-model.html) I noticed this.  

Is there a different way lesson authors should denote code in exercise blocks? or is this an acceptable low contrast? (the exercise block background is the same color as a code block background) 

[linking to review issue for future reference](https://wave.webaim.org/report#/https://carpentries-incubator.github.io/deep-learning-intro/3-monitor-the-model.html)

**Comment by tobyhodges** [comment]
I can confirm that, based on https://coolors.co/contrast-checker/d63384-f8f9f9, this falls short of the 4.5:1 ratio required for [the WCAG AA success criterion](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). The code-formatted text on pure white background meets that ratio exactly (see https://coolors.co/contrast-checker/d63384-ffffff).

Can we adjust the colour for inline code to keep a high contrast on this grey background, e.g. to [#A50D5E](https://coolors.co/contrast-checker/a50d5e-f7f8f8)?

**Comment by ErinBecker** [comment]
Thanks for this issue @brownsarahm. The @carpentries/workbench-maintainers started to discuss this at our meeting today. Here's what we've tracked down so far: 

Kelly and I started by inspecting the code for in-line code within an exercise block. The color for the in-line code appears to come from the `_rfs.scss` stylesheet (line 294). 

<img width="1240" alt="Screenshot 2023-11-03 at 8 52 51 AM" src="https://github.com/carpentries/workbench/assets/19176319/57076d9f-c989-4195-bea7-982c9e9d3b6b">

This sylesheet appears to be [imported from Bootstrap](https://github.com/twbs/bootstrap/blob/main/scss/vendor/_rfs.scss), but line 294 of this style sheet appears to be about resizing safari iframes. We have no idea how this is related to setting text color. 


<img width="762" alt="Screenshot 2023-11-03 at 9 27 03 AM" src="https://github.com/carpentries/workbench/assets/19176319/77fc2f72-e43b-4bee-b553-500b5de69d6c">


Varnish's `styles.css.scss` imports `codeblocks.scss`, which includes a definition of a `code` element starting on line 46. Line 53 sets the color property for the code element to `inherit`. 


<img width="721" alt="Screenshot 2023-11-03 at 9 03 24 AM" src="https://github.com/carpentries/workbench/assets/19176319/35237bad-94d8-4c21-9b6e-18eb2243fcdb">


We found a [tutorial video about css cascading](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance) that seems relevant, but haven't yet been able to track down where the code element is inheriting its color from. We are pretty confident we can override this inhertance by defining code color somewhere, but we don't know where it should be defined.


We'll be checking in with the other Workbench Maintainers next week, who know a lil' more about css (😂) and hopefully figure out where this property needs to be set. This is turning into a great learning opportunity for us to understand how the Workbench uses css.

**Comment by tobyhodges** [comment]
Note that the issue is with contrast of inline code (i.e. monospaced text within a paragraph of normal text), rather than code in codeblocks. e.g. `code {color: #a50d5e;}` rather than

```
code {
    color: #a50d5e;
}
```

So `codeblocks.scss` may not be the correct location for making a change like this. Having said that, I am not sure where else to suggest! But it could be an important distinction.

**Comment by froggleston** [comment]
Hi all. The CSS of the code block declaration in bootstrap is overridden by the syntax-highlighting.scss class in varnish. @ErinBecker is correct that inheritance is in play here, but as the syntax-highlighting.scss inherits from the bootstrap code class, but *doesn't* specify its own color, then it will inherit the bootstrap color. We can override this by adding a specific color property in the syntax-highlighting.scss class: 
![image](https://github.com/carpentries/workbench/assets/1872302/6604db92-7da5-485e-8ab7-5d95151da34a)

**Comment by froggleston** [comment]
Many thanks for raising this @brownsarahm - @zkamvar and I went through this today, and we agreed that setting the background-color to white for the inline code should increase the contrast. I've implemented a fix and it will be in the upcoming varnish 0.3.2 release.

**Comment by zkamvar** [comment]
Just a note, [Rob created the release 20 minutes ago](https://github.com/carpentries/varnish/releases/tag/0.3.2) and it is now live on the R universe: https://carpentries.r-universe.dev/varnish
